src/components/messenger/list/rewards-tooltip/container.tsx

```diff
@@ -0,0 +1,44 @@
+import * as React from 'react';
+
+import { connectContainer } from '../../../../store/redux-container';
+import { RootState } from '../../../../store/reducer';
+import { rewardsTooltipClosed } from '../../../../store/rewards';
+
+import { RewardsTooltip } from '.';
+import { Container as RewardsItemContainer } from '../../../settings-menu/rewards-item/container';
+export interface PublicProperties {}
+
+export interface Properties extends PublicProperties {
+  meowPreviousDayInUSD: string;
+  isLoading: boolean;
+
+  rewardsTooltipClosed: () => void;
+}
+
+export class Container extends React.Component<Properties> {
+  static mapState(state: RootState): Partial<Properties> {
+    const { rewards } = state;
+    return {
+      meowPreviousDayInUSD: RewardsItemContainer.totalPriceInUSD(rewards.meowPreviousDay, rewards.meowInUSD),
```

Comment:
Don't reach into another component. It may work but those aren't meant to be shared.

Reply:
ok